### ISSUE TITLE
401 Response when sending requests to external API through proxy

### ISSUE THREAD

**Issue by mat-burke** [commercetools/merchant-center-application-kit#1103]
Related Support Ticket:
https://jira.commercetools.com/browse/SUPPORT-6998

We developed and added a user application with requests to our server through your proxy, according to the documentation https://appkit.commercetools.com/main-concepts/proxy-to-external-api

But when we try send requests we get 401 error. Could you explain what can be reason?

I am sending this request

{ method: "GET" credentials: "include" headers: {Accept: "application/json" X-Forward-To: "https://commercetools-dev.vertexconnectors.com/connector-config"}
}
to the https://mc-api.commercetools.com/proxy/forward-to

Token managing by ApplicationShell React component from '@commercetools-frontend/application-shell'

CSP is configured.

**Comment by emmenko** [comment]
I answered in the ticket.

TL;DR: the app runs on GCP-US but you're trying to connect to the MC HTTP API on GCP-EU

**Comment by SerhiiMykhaliuk** [comment]
@emmenko Now we have another case, like describe in the Support Ticket: 
We are sending this request:
{ method: "GET" credentials: "include" headers: {Accept: "application/json" X-Forward-To: "https://commercetools-dev.vertexconnectors.com/connector-config"}
}
to the https://mc-api.commercetools.com/proxy/forward-to

And we get 204 Error now

**Comment by emmenko** [comment]
`204` is not an error, it's the response for the preflight request for CORS (`OPTIONS`)

https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/OPTIONS

**Comment by SerhiiMykhaliuk** [comment]
> I answered in the ticket.
> 
> TL;DR: the app runs on GCP-US but you're trying to connect to the MC HTTP API on GCP-EU

It was really helpful, thank you.

**Comment by mat-burke** [comment]
Thanks! I'm closing this one.
